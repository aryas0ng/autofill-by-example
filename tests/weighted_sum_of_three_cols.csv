1,0,9,20
2,4,8,24
2,3,7,21
3,8,6,26
10,10,5,
5,5,4,
4,4,3,
19,4,2,
2,21,1,
1,21,0,
9,10,4,
7,7,3,
4,4,2,
3,3,1,
12,3,0,
4,16,1,
1,16,0,
26,27,5,
8,8,4,
3,3,3,
3,3,2,
8,11,1,
4,4,0,