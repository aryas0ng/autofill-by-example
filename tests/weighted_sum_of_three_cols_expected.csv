1,0,9,20
2,4,8,24
2,3,7,21
3,8,6,26
10,10,5,40
5,5,4,23
4,4,3,18
19,4,2,46
2,21,1,27
1,21,0,23
9,10,4,36
7,7,3,27
4,4,2,16
3,3,1,11
12,3,0,27
4,16,1,26
1,16,0,18
26,27,5,89
8,8,4,32
3,3,3,15
3,3,2,13
8,11,1,29
4,4,0,12